lib/component.js

```diff
@@ -172,6 +172,16 @@
     // Use `nextProps` or `component.props` and grab `model` and `collection`
     // from there
     var props = nextProps || component.props || {};
+    if (component.overrideModel && typeof component.overrideModel === 'function'){
```

Comment:
Sounds good. Let me know when you're done so I can share it. You don't need a lot of followers, just tag #react and #bootrstrap and #react-router.

P.s. you have one new follower that you can retweet ;)